loops-and-arrays.js

```diff
@@ -40,7 +55,15 @@ const numbers = [2, 4, 5, 29, 38];
 // Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 getallen zou bevatten!
 // ==========================================
 
-const squares = [30, 2, 8, 24, 11];
+//const squares = [30, 2, 8, 24, 11];
+//console.log(squares);
+
```

Comment:
Dit was de eerste oplossing (nu in commentaren), hieronder de echte oplossing

Reply:
Dat je dit meld maakt voor mij ook even duidelijk wat je bedoelt. Top!